apps/src/pythonlab/layout/HorizontalLayout.tsx

```diff
@@ -41,6 +42,7 @@ const HorizontalLayout: React.FunctionComponent = () => {
     axis: 'x',
     initial: INITIAL_INFO_PANEL_WIDTH,
     min: MIN_LEFT_PANEL_WIDTH,
+    onResizeStart: () => logOnResize('pythonlab'),
```

Comment:
Curious if it would be of interest to include whether they resized in horizontal vs vertical layout.

Reply:
I'll ask Sam!

Reply:
added more metadata!